build-logic/jvm/src/main/kotlin/build-logic.build-info.gradle.kts

```
@@ -0,0 +1,14 @@
+import buildlogic.BuildInfoTask
+
+plugins {
+    `java`
+}
+
+val generateBuildInfoTask by tasks.register("generateBuildInfo", BuildInfoTask::class.java) {
```

It should be `registering`, not `register`

```suggestion
val generateBuildInfoTask by tasks.registering(BuildInfoTask::class) {
```

---

ah, so I'll also have to change how it's resolved later in the sigstore-java/build.gradle.kts